core/src/service/import_export_service.rs

```diff
@@ -0,0 +1,140 @@
+use crate::model::client_conversion;
```

Comment:
This code is good and fine, add some light integration tests in `tests/`.

Maybe one that takes content on disk, imports it, verifies some of the closure properties, exports it out somewhere else, and verify that it's what you expect.

Reply:
Also, isn't there code in CLI that can be deleted now? Can't it use this stuff?

Reply:
You're right, ill make it use it.